.github/workflows/setup_rh_config/action.yaml

```diff
@@ -27,3 +27,4 @@ runs:
         echo "token: ${{ inputs.token }}" >> ~/.rh/config.yaml
         echo "username: ${{ inputs.username }}" >> ~/.rh/config.yaml
         echo "api_server_url: ${{ inputs.api_server_url }}" >> ~/.rh/config.yaml
+        echo "autosave: false" >> ~/.rh/config.yaml
```

Comment:
Just a question: could it make sense to have autosave possibly be an array of types to autosave? e.g. `['cluster', 'module']` or is that too much control? Thinking of cases where some people might want to autosave functions as well, or maybe autosave everything (envs, ...)?

Reply:
for envs i think less relevant, bc those get saved automatically as subresources of functions or modules, maybe clusters could be treated differently, but i think that's adding more complexity than we need for now unless we get feedback about having more fine grained control over the resource types

Reply:
yeah i agree that we can jsut leave it as a generic param